R/autoplot.R

```diff
@@ -23,21 +23,24 @@
 #' @author Ari Friedman, Olaf Mersmann
 autoplot.microbenchmark <- function(object, ...,
                                     log=TRUE,
-                                    y_max=1.05 * max(object$time)) {
+                                    y_max=NULL) {
   if (!requireNamespace("ggplot2"))
     stop("Missing package 'ggplot2'.")
   y_min <- 0
   object$ntime <- convert_to_unit(object$time, "t")
+  if (is.null(y_max)) {
+    y_max <- max(object$ntime)
```

Comment:
Is there a reason you removed the 1.05 multiplier that was the default?

Reply:
I mentioned it in the cm, but I should have explained it better. The multiplier had no effect, as coord_flip() was called without ylim and therefore used the ggplot default, which is the range of values without any multiplier: `ylim = range(object$ntime)`. Ggplot uses some padding between the "active" plot area and the frame anyway. So the bottom line is that _not_ removing the multiplier would change the behaviour of autoplot.microbenchmark (add more padding than before) when called without any parameters other than `object`.

Reply:
Got it. Thanks for (re-)explaining!